e2e-tests/content-strings/fields/insurance/exporter-business/index.js

```diff
@@ -78,7 +78,24 @@ export const EXPORTER_BUSINESS_FIELDS = {
       TITLE: 'UK telephone number (optional)',
     },
   },
-
+  CONTACT: {
+    [FIELD_IDS.INSURANCE.EXPORTER_BUSINESS.CONTACT.COMPANY_NAME]: {
+      LABEL: 'Your company name',
+      HINT: 'This information comes from Companies House',
+    },
+    [FIELD_IDS.INSURANCE.EXPORTER_BUSINESS.CONTACT.FIRST_NAME]: {
+      LABEL: 'First name',
+    },
+    [FIELD_IDS.INSURANCE.EXPORTER_BUSINESS.CONTACT.LAST_NAME]: {
+      LABEL: 'Last name',
+    },
+    [FIELD_IDS.INSURANCE.EXPORTER_BUSINESS.CONTACT.EMAIL_ADDRESS]: {
+      LABEL: 'Email address',
+    },
```

Comment:
I think it would better to have generic fields for these in the root of insurance `FIELD_IDS` and then they can be used throughout, what do you think?

Reply:
I changed to use the FIELD_IDs imported from account.  Is that fine?